### DISCUSSION TITLE
Send Invoce via XML

### DISCUSSION THREAD

**Discussion by imbundle** [fattureincloud/api#457] [Q&A] [answered]
Buon pomeriggio avrei un esigenza di questo tipo.
Ho un gestionale che mi fornisce già un XML valido da mandare via PEC.
Io però vorrei usare le vostre API per mandare la fattura a fatture in cloud e poi mandarla via SDI channel
Tra la lista delle API non ho visto un API in post con un file XML

**Comment by valmoz** [comment, marked as answer]
Buongiorno @imbundle 
abbiamo già risposto alla sua domanda in questa discussione: https://github.com/fattureincloud/api/discussions/436

Grazie
Mauro

**Reply by imbundle** [reply]
Capisco un vero peccato non poter importare un XML

**Reply by valmoz** [reply]
@imbundle capisco.
Come alternativa le posso solamente suggerire di creare un'applicazione custom che riceva in ingresso l'XML per trasformarlo nel JSON necessario alle nostre API per importare il documento.
Di seguito una guida: [Invoice Creation](https://developers.fattureincloud.it/docs/guides/invoice-creation/)

Grazie
Mauro
